testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -84,35 +88,35 @@ protected BlackBoxBoundedContext(boolean multitenant, Enricher enricher) {
     }
 
     /**
-     * Creates a single tenant bounded context with the default configuration.
+     * Creates a single-tenant instance with the default configuration.
      */
-    public static SingleTenantBlackBoxContext newInstance() {
+    public static SingleTenantBlackBoxContext singleTenant() {
```

Comment:
Well, I agree. But I have specifically asked to leave `newInstance` instead of `singleTenant` to keep our API compatible with the existing testing code and ease the migration. The reason is that we are `1.0.0-SNAPSHOT` and each published build breaks projects which reference it immediately — if an API element like this one is changed. This is the case for at least `users` library and one more proprietary project.

What if we introduce `singleTenant()`, keep `newInstance()` as an alias and mark the latter as `@Deprecated` to remove in `pre2`? In this way we'll still be able to compile our code (i.e. `users` library) and schedule a migration once we are ready to do it.

Reply:
Did so.